Marketplace/dotnet/Marketplace/Pages/Index.cshtml

```diff
@@ -5,10 +5,10 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Zwap store!</h1>
+    <h1 class="display-4">Integrations!</h1>
     @if (User.Identity.IsAuthenticated)
     {
-        <iframe id="zsiFrame" src="@(Model.IFrameUrl)" height="600px" width="100%" style="border: 0;"></iframe>
+        <iframe id="zsiFrame" src="@(Model.IFrameUrl)" height="700px" width="1200px" style="border: 0;"></iframe>
```

Comment:
How does it look on smaller screens (mobile and tablet)?

Reply:
Terrible, but it's just an example.